tests/lib/rules/jsx-child-element-spacing.js

```diff
@@ -131,66 +131,87 @@ ruleTester.run('jsx-child-element-spacing', rule, {
   }],
 
   invalid: [{
-    code: `
-      <App>
-        foo
-        <a>bar</a>
-      </App>
-    `,
+    code: `<App>
```

Comment:
it is related in the sense that I wanted to remove extraneous whitespace from the test cases, so that the loc test assertions were more predictable and were not affected by the indentation of the surrounding code

Reply:
having ```code: `\n``` seems like it achieves that?

Reply:
I'm not positive I understand, but I updated this to make it clearer. the indentation is still flush with column 1 so that the column counts are fixed, but the code samples now begin with a newline which may help with readability

Reply:
Thanks, makes sense :-) rebased, will merge soon.